### PR TITLE
Update pr template

### PR DESCRIPTION
**Pull request by talnor**
# Description
* Update PR template
* black

_The features of this PR primarily concerns end-users/bioinformaticians/internals_

_Summary of the changes made:_

_If not self-evident, mention what prompted the change._

## Primary function of PR
- [ ] Hotfix
- [x] Patch
- [ ] Minor functionality improvement
- [ ] New type of analysis
- [ ] Backward-breaking functionality improvement
- [ ] This change requires internal documents to be updated
- [ ] This change requires another repository to be updated

# Testing
_If the update is a hotfix, it is sufficient to rely on the development testing along with the Travis self-test automatically applied to the PR._

_Test routine to verify the stability of the PR:_
- _`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-microsalt-stage.sh update_PR_template`_
- _`us`_
- _`source activate S_microSALT`_
- _(SITUATIONAL) `export MICROSALT_CONFIG=/home/proj/dropbox/microSALT.json`_
- _Select a relevant subset of the following:_
- _`microSALT analyse project MIC3109`_
- _`microSALT analyse project MIC4107`_
- _`microSALT analyse project MIC4109`_
- _`microSALT analyse project ACC5551`_

_Verify that the results for projects MIC3109, MIC4107, MIC4109 & ACC5551 are consistent with the results attached to AMSystem doc 1490, Microbial_WGS.xlsx_

## Test results
_These are the results of the tests, and necessary conclusions, that prove the stability of the PR._

# Sign-offs
- [ ] Code tested by @octocat
- [ ] Approved to run at Clinical-Genomics by @talnor

### CODE REVIEW

**Review by sylvinite — APPROVED** [review]

**Comment by sylvinite** [comment]
iirc there is also a way to auto-black using github actions. Might save some hassle in the future.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -20,15 +20,26 @@ _If not self-evident, mention what prompted the change._
...
+  - _`us`_
+  - _`source activate S_microSALT`_
+  - _(SITUATIONAL) `export MICROSALT_CONFIG=/home/proj/dropbox/<your_new_microSALT_config>.json`_
+  - _`microSALT analyse /home/proj/stage/microbial/queries/merrymink.json --input /home/proj/stage/microbial/fastq/merrymink/`_
+- or: Run microSALT with cg:
+  - _`us`_  
+  - _(SITUATIONAL) `export MICROSALT_CONFIG=/home/proj/dropbox/<your_new_microSALT_config>.json`_    
+  - _`cg workflow microsalt start merrymink`_
 
 _Verify that the results for projects MIC3109, MIC4107, MIC4109 & ACC5551 are consistent with the results attached to AMSystem doc 1490, Microbial_WGS.xlsx_
```
[the remark this answers is not in the document]
Comment (talnor) [reply .github/pull_request_template.md:44]: True, that line should be clarified and updated. merrymink is the same as ACC5551, but does not contain the others.

File: microSALT/__init__.py [hunk 2]
```diff
@@ -119,8 +122,10 @@
                                 )
                                 output, error = proc.communicate()
                                 if proc.returncode != 0:
-                                  logger.error("Database writing failed! Invalid user access detected!")
-                                  sys.exit(-1)
+                                    logger.error(
+                                        "Database writing failed! Invalid user access detected!"
+                                    )
```
[the remark this answers is not in the document]
Comment (talnor) [reply microSALT/__init__.py:127]: Interesting! Or do we just let black do its thing?

**Comment by talnor** [comment]
> iirc there is also a way to auto-black using github actions. Might save some hassle in the future.

That would be super! I'll make an issue for that. We could add this to all the microbial workflows when we have figured out how to set it up.

File: microSALT/__init__.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (sylvinite) [reply microSALT/__init__.py:127]: Personally I think seperating a parenthesis like this negatively affects readability as it doesn't make the code easier to read; but steals valuable screen space. It also introduces an additional layer of indentation which negatively affects the code legibility.

Anyway. It's not a hill I'm dying on. Just informing that black can be configed, see https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-via-a-file

**Comment by talnor** [comment]
Closing stale PR.

**Pull request closed without merging**